heidrun/ia.rb

```diff
@@ -0,0 +1,238 @@
+# coding: utf-8
+
+# TODO: this list is from the old system and is likely out of date
```

Comment:
This list is up to date, to the best of my knowledge. At some point we may want to refactor this to specify it in the profile for IA once we've established the model and UI.  cc @guegueng